tests/d3d12_render_target.c

```diff
@@ -2139,3 +2139,226 @@ void test_mismatching_rtv_dsv_size(void)
 
     destroy_test_context(&context);
 }
+
+
+void decode_rgb9e5(uint32_t packed, float color[3]) {
```

Comment:
Lots of brace placement nits here.

Reply:
derp, i was half asleep when writing this. will fix later.